HOW_TO_RELEASE.md

```diff
@@ -2,78 +2,90 @@
 
 Time required: about an hour.
 
+These instructions assume that `upstream` refers to the main repository:
+```
+$ git remote -v
+{...}
+upstream        https://github.com/pydata/xarray (fetch)
+upstream        https://github.com/pydata/xarray (push)
+```
+
+ 0. Write a release summary: ~50 words describing the high level features. This
+    will be used in the release emails, tweets, GitHub release notes, etc. 
+    I (@max) find it helpful to use a PR given I'm not always familiar with all the
+    features of the release, and the summary is distributed widely.
```

Comment:
I think it definitely makes sense to get a review of the summary. Also, I'd remove this: there's no need to justify yourself for that suggestion :smile:

In any case, I'd like to suggest writing the summary as item 2, then opening a PR with the contributors, the summary and the `whats-new.rst` fixes (new item 4) as item 5 to get a chance for a review and once that is merged continue with the current item 4 (after syncing again).

So now we'd have:
1. sync branch
2. write summary
3. get the contributors (also: what should happen in the rare case that the one releasing is not included in that list?)
4. check `whats-new.rst` and the docs
5. open a PR to add items 2 to 4 to `whats-new.rst`
6. resync (should that be a new step or should that also be in item 5?)
7. continue with the current item 4

Does that make sense?